third_party/icu/make_data_assembly_for_skiko.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python2
+
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import binascii
+import optparse
+import sys
+
+parser = optparse.OptionParser()
+parser.add_option("--mac",
+                  help="generate assembly file for Mac/iOS (default: False)",
+                  action="store_true", default=False)
+parser.add_option("--win",
+                  help="generate assembly file for Windows (default: False)",
+                  action="store_true", default=False)
+parser.set_usage("""make_data_assembly  icu_data [assembly_file] [--mac] [--win]
+    icu_data: ICU data file to generate assembly from.
+    assembly_file: Output file converted from icu_data file.""")
+(options, args) = parser.parse_args()
+
+if len(args) < 1:
+  parser.error("ICU data file is not given.")
+
+input_file = args[0]
+n = input_file.find(".dat")
+if n == -1:
+  sys.exit("%s is not an ICU .dat file." % input_file)
+
+if len(args) < 2:
+  output_file = input_file[0:n] + "_dat.S"
+else:
+  output_file = args[1]
+
+if input_file.find("l.dat") == -1:
+  if input_file.find("b.dat") == -1:
+    sys.exit("%s has no endianness marker." % input_file)
+  else:
+    step = 1
+else:
+  step = -1
+
+input_data = open(input_file, 'rb').read()
+n = input_data.find(b'icudt')
+if n == -1:
+  sys.exit("Cannot find a version number in %s." % input_file)
+
+version_number = "_skiko" + input_data[n + 5:n + 7].decode("ascii")
```

Comment:
this is the only difference with the original script

Reply:
Let's make a comment about this in this file. And point to the original `../externals/icu/scripts/make_data_assembly.py`